ScratchWikiSkin.skin.php

```diff
@@ -49,15 +49,21 @@ class SkinScratchWikiSkin extends SkinTemplate {
 		$template = 'ScratchWikiSkinTemplate', $useHeadElement = true;
 
 	/**
-	 * Add CSS via ResourceLoader
+	 * This function adds JavaScript via ResourceLoader
 	 *
-	 * @param $out OutputPage
+	 * Use this function if your skin has a JS file(s).
+	 * Otherwise you won't need this function and you can safely delete it.
+	 *
+	 * @param OutputPage $out
 	 */
-	public function setupSkinUserCss( OutputPage $out ) {
-		parent::setupSkinUserCss( $out );
+
+	public function initPage( OutputPage $out ) {
+		parent::initPage( $out );
+		$out->addModules('skins.scratchwikiskin2.js');
```

Comment:
Can you please stop changing things that weren't part of the original issue? There is no such module as `skins.scratchwikiskin2.js`.
```suggestion
```